src/main.js

```diff
@@ -432,6 +432,10 @@ function handleAppWebContentsCreated(dc, contents) {
       log.info(`Untrusted popup window blocked: ${url}`);
       return;
     }
+    if (isKnownTeam(url)) {
```

Comment:
This logic doesn't make sense to me. The if-block right before this is checks `!isTrustedURL(url)`, but since `isTrustedURL` just calls `isKnownTeam`, this code block seems to work like
```javascript
if (!isKnownTeam(url)) {
    return;
}
if (isKnownTeam(url)) {
    return;
}
```
When is it supposed to continue past both of these if statements, and does it actually work in that case?

Reply:
yeah, good catch, I totally misread a part